### PR TITLE
Require periodical builds

### PR DESCRIPTION
**Pull request by vladfaust**
Closes https://github.com/veelenga/awesome-crystal/issues/347.

The community may want to add other CI services. I use Travis CI only.

### CODE REVIEW

**Comment by vladfaust** [comment]
:ping_pong:

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Keep in mind that the list is very mature by now, so the bar of getting somethin
     - Or, for finished projects, issues and pull requests are responded to
   - Ensure your work follows general quality rules
   (look at [Writing Shards](https://crystal-lang.org/docs/guides/writing_shards.html) tutorial if this is a Crystal shard)
+  - Your shard **must** have periodical (e.g. daily or weekly) builds set up on a publicly accessible CI against latest stable Crystal (nightly is optional)
+    - For Travis CI see https://docs.travis-ci.com/user/cron-jobs. Nightly builds are [broken](https://github.com/travis-ci/travis-ci/issues/10126) at the moment, so you may want to use [docker-based builds](https://gist.github.com/vladfaust/9cf8ca03b00a3ab5c0e86ca8381acad7)
```
[the remark this answers is not in the document]
Comment (vladfaust) [reply .github/CONTRIBUTING.md]: That info wouldn't become stale, because this docker approach is absolutely valid even if nightly fixed on Travis. I see no reason to remove a helpful information :thinking: 

I could mention this PR in the issue and when it's fixed, someone (hopefully me) will send a PR here to remove these "nightly is currently broken" words.

[the remark this answers is not in the document]
Comment (veelenga) [reply .github/CONTRIBUTING.md]: I understand. But still, I don't want to add information that can become stale soon and rely on "somebody will ping/fix it". If you think this information is necessary here, let's rephrase it to something like this:

"As an option, you can run nightly Crystal builds using docker-based-builds"

[the remark this answers is not in the document]
Comment (vladfaust) [reply .github/CONTRIBUTING.md]: Nice point. Okay, gonna apply your changes
